conf/streamline-dev.yaml

```diff
@@ -88,6 +88,9 @@ storageProviderConfiguration:
 #    dataSource.url: "jdbc:mysql://localhost/streamline_db"
 #    dataSource.user: "streamline_user"
 #    dataSource.password: "streamline_password"
+#  db.migrate:
```

Comment:
I dont think we should expose migration specific options in the config files. Users can mess up. These can be command line options to bootstrap or hard code to some reasonable values.